avocado/core/nrunner/runnable.py

```diff
@@ -309,8 +309,9 @@ def _module_exists(module_name):
         mod_path = os.path.join('plugins', 'runners', module_filename)
         return pkg_resources.resource_exists('avocado', mod_path)
 
-    def pick_runner_command(self, runners_registry=None):
-        """Selects a runner command based on the runner.
+    @staticmethod
+    def pick_runner_command(kind, runners_registry=None):
+        """Selects a runner command based on the runner kind.
 
         And when finding a suitable runner, keeps found runners in registry.
```

Comment:
IMO, if we are really converting this into a static method, this docstring must be updated accordingly.